### DISCUSSION TITLE
第一个发个言

### DISCUSSION THREAD

**Discussion by wgzhong** [mlc-ai/mlc-zh#1] [General]
抱紧大佬的腿，希望课程有记录，通过课程我希望能学习到如何添加一个新的硬件比如npu（rk1808这种的）。

**Comment by chenge191** [comment]
不知道视频啥时候出来

**Reply by Hzfengsy** [reply]
最早一期会在美国时间6.17出来（B站上传可能会有延迟）。详见 https://mlc.ai/summer22-zh/schedule

**Comment by wikipedia2008** [comment]
思远博士，我要跟你学习，可以吗
